TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/EntityManagerFactory.php

```diff
@@ -146,7 +152,7 @@ public function create()
         if (isset($this->settings['doctrine']['filters']) && is_array($this->settings['doctrine']['filters'])) {
             foreach ($this->settings['doctrine']['filters'] as $filterName => $filterClass) {
                 $config->addFilter($filterName, $filterClass);
-                $entityManager->getFilters()->enable($filterName);
+                $this->enabledFilterInstances[$filterName] = $entityManager->getFilters()->enable($filterName);
```

Comment:
After working again a bit on #462 I noticed one thing: there might be cases where some custom code adds filters outside the configured filters in `Settings.yaml`, at least once that change gets merged. Hence, it is/becomes unreliable to keep track of the enabled filters ourself.

Luckily, Doctrine already provides an API for this:
`$entityManager->getFilters()->getEnabledFilters();`